### PR TITLE
Add exercise: etl

### PR DESCRIPTION
**Pull request by aawray**

### CODE REVIEW

**Comment by andrej-makarov-skrt** [comment]
Not sure if input and output must be Dicts. @SaschaMann, is this example okay?

**Comment by SaschaMann** [comment]
@andrej-makarov-skrt I'll take a look in a bit, I was busy over the weekend.

**Comment by SaschaMann** [comment]
@andrej-makarov-skrt I think Dicts make more sense for out- and input here. Also Chars instead of 1-letter strings.

**Comment by aawray** [comment]
@SaschaMann changed test sets and example, please check it out.

File: exercises/etl/runtests.jl [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+end
+
+@testset "multiple scores with multiple letters" begin
+    input = Dict(1=>["A", "E"], 2=>["D", "G"])
+    output = Dict("g"=>2, "e"=>1, "a"=>1, "d"=>2)
+    @test transform(input) == output
+end
+
+@testset "multiple scores with differing numbers of letters" begin
+    input = Dict(1=>[ "A", "E", "I", "O", "U", "L", "N", "R", "S", "T" ],
```
Comment (SaschaMann) [inline exercises/etl/runtests.jl]: There are only single letters as key dictionaries, so they should be Chars 'A' etc. instead of Strings

Comment (aawray) [inline exercises/etl/runtests.jl]: Done.

**Comment by SaschaMann** [comment]
That works, thanks for adding this exercise! :)

**Pull request merged**